os/Kconfig

```diff
@@ -17,6 +17,10 @@ config LIBDIR
 	string
 	option env="LIBDIR"
 
+config LOADABLEDIR
```

Comment:
To be clear, this commit introduces CONFIG_LOADABLEDIR in addition to CONFIG_APPS_DIR and apps inside CONFIG_LOADABLEDIR dir alone will be loadable. Is it right?

Reply:
LOADABLEDIR and APPDIR are different.
LOADABLEDIR for loadable_apps folder
APPDIR for apps folder

They are same level of folder, not subdirectory.